lib/rouge.rb

```diff
@@ -2,6 +2,15 @@
 
 # stdlib
 require 'pathname'
+require_relative 'rouge/version'
```

Comment:
I think `require_relative` is okay, for now. @jneen what are your thoughts on this?

On Aug 15, 2015, at 5:56 PM, "Patrik Ragnarsson" <notifications@github.com> wrote:

> In lib/rouge.rb:
> 
> > @@ -2,6 +2,15 @@
> >  
> >  # stdlib
> >  require 'pathname'
> > +require_relative 'rouge/version'
> Maybe the intent is more clear with require_relative
> …
> —
> Reply to this email directly or view it on GitHub.
>